### ISSUE TITLE
Create "inactive chapter" notification or page

### ISSUE THREAD

**Issue by kstack7** [girldevelopit/gdi-website#464]
**Description**
_We need to add a page for inactive chapters that provides information for those hoping to jumpstart the chapter again._

**Ask**
_Create a notification or a redirect page that has contact information or "next steps."_

**To Do**
- [ ] Discuss the best approach with CLs and HQ
- [ ] Create new page (if that direction) and finalize content
- [ ] Redirect users to that page when visiting inactive chapters

**Additional Info**
_Need to decide if we want to indicate that the chapter is inactive in the /chapters page or leave the chapter but redirect when clicking. Good starting point for discussion._

**Comment by kstack7** [comment]
@corinnepw

**Comment by corinnepw** [comment]
@kstack7 this is awesome! Did we talk about this yet? LaVonna and I were just discussing this a week or so ago. Would love to work on getting some language and a form together. We have the list of "inactive chapters"

**Comment by kstack7** [comment]
@corinnepw no specifics yet. Just recalling the initial suggestion and then we discussed it in yesterday's meeting real quick. Should be straight-forward as long as we have the content we need :+1:

**Comment by Roenok** [comment]
Here's my idea, complete with 12-second mockup:

If someone visits an inactive chapter, the page redirects them to the regular chapters page, with an alert message on top. The alert says something like "We're sorry to say that CHAPTER NAME is currently inactive. Are you interested in helping us bring it back? [Fill out our interest form](https://docs.google.com/forms/d/18vn6QF64wg8i7euJXSA4HlYWO4AlL9i04o7057-ewuQ/viewform)"

![image](https://cloud.githubusercontent.com/assets/925963/16245458/02383ff6-37cf-11e6-8d4a-6a4f1567c1e0.png)

**Comment by kstack7** [comment]
Yeah this is a good direction, @Roenok. Wouldn't hurt to revisit the notification anyways since I always thought the notifications could use some love. Little too subtle in some cases.

**Comment by kstack7** [comment]
Happy to take a look at this next once we have content/ideas, ladies :)

**Comment by kstack7** [comment]
So I started to focus on this, but realized we'll need to discuss the map/layout/chapter count with inactive chapters in mind. Right now the site only pulls active chapters, so you'd never be able to navigate to the inactive ones.

Do we let users navigate to the chapter and then give a notification/warning? Or block navigation to the chapter with a redirect back to the chapters list (notification/warning too). I'll change the logic once this is decided.

Also would need final content/sentence for the warning :)

**Comment by Roenok** [comment]
@kstack7 Whenever we get around to this one, here is my idea:
- Inactive chapters are not on the map or the chapter list
- Inactive chapters don't have a visible individual page
- Inactive chapters do have a "stub" page that's just a redirect to the main chapters page, with the alert. 

Here's our user stories:
- Susan goes to the GDI homepage, looking for GDI Big Town. GDI Big Town is inactive, so it's not on the map or the list of chapters. She sees the link "Don't see your hometown? Start a new chapter!" prominently featured on both pages and submits the interest form. 
- Kamala googles "GDI Big Town" and finds an old link to the individual chapter page. She clicks on it. Instead of being taken to an empty page with no real content, she is redirected to the chapter page with the alert message. She can either check her state for a nearby chapter, or click the link in the alert message to fill out the interest form.

**Comment by luizfnunesmarques** [comment]
Is this issue is still open? Considering the last specs from @Roenok 
cc @kstack7

**Comment by luizfnunesmarques** [comment]
up.

**Comment by kstack7** [comment]
Hi @luizfnunesmarques -- this issue is still on hold until we get the green light from HQ :) Thanks for you interest!
